base/src/main/scala/co/uproot/abandon/Config.scala

```diff
@@ -96,6 +101,14 @@ object SettingsHelper {
     }
   }
 
+  def makeDateConstraints(config: Config): DateConstraint = {
+    val format = DateTimeFormatter.ISO_LOCAL_DATE
+
+    val from = Try(LocalDate.parse(config.getString("from"), format)).map(Some(_)).getOrElse(None)
+    val to = Try(LocalDate.parse(config.getString("to"), format)).map(Some(_)).getOrElse(None)
```

Comment:
For parsing the constraint dates, you could use the same date-parser as used in the `.ledger` parser. So the date format will match the one in the `.ledger`.